dashboard/forms.py

```diff
@@ -190,6 +217,14 @@ class Meta:
             'date': SelectDateWidget(),
         }
 
+    def clean(self):
```

Comment:
this is repeated with pnm form.  you might want to make a parent class for events like `EventForm` or something like that so this clean function doesnt get repeated a bunch of times